test/cppsuite/test_harness/runtime_monitor.cxx

```diff
@@ -338,4 +351,29 @@ runtime_monitor::finish()
           "limits.");
 }
 
+/*
+ * This function generates a file that contains the values of the different statistics that need to
+ * be saved as indicated by the configuration file.
+ */
+void
+runtime_monitor::save_stats(const std::string &filename)
+{
+    std::string stat_info = "[{\"info\":{\"test_name\": \"" + filename + "\"},\"metrics\": [";
+
+    for (const auto &stat : _stats) {
+        if (stat->get_save())
+            stat_info += "{\"name\":\"" + stat->get_name() +
+              "\",\"value\":" + stat->get_value_str(_cursor) + "},";
+    }
+
+    /* Remove last extra comma. */
+    if (!stat_info.empty()) {
```

Comment:
I will need to fix an error here as it is always true, thanks @ajmorton for spotting it.

Reply:
Fixed in 2251c4838de185c84ab00cf4485daf64c137da8f